### ISSUE TITLE
Have problem while use metaboss to change metadata, 

### ISSUE THREAD

**Issue by PrimalCat-Real** [samuelvanderwaal/metaboss#32]
I tried to update metadata but have this problem

```
Using a public RPC URL is not recommended for heavy tasks as you will be rate-limited and suffer a performance hit.
            Please use a private RPC endpoint for best performance results.
Error: invalid variant
```

As i understand I must add --rpc but now i have another error message

```
error: Found argument '--rpc' which wasn't expected, or isn't valid in this context

USAGE:
    metaboss-windows-latest.exe update data --account <account> --keypair <keypair> --new-data-file <new-data-file>

```
How use private rpc to update metadate?

**Comment by samuelvanderwaal** [comment]
You can still use a public RPC it's just going to be a lot slower if you're updating a lot of metadata files. What RPC did you try to use initially that gave you an error?

For specifying a private RPC endpoint you need to pass the option directly to the `metaboss` command. In your case:

```
metaboss-windows-latest.exe --rpc <YOUR_RPC_URL> update data --account <account> --keypair <keypair> --new-data-file <new-data-file>
```

**Comment by samuelvanderwaal** [comment]
@PrimalCat-Real this should be resolved in v0.3.1. You can re-open the issue if you experience it again on the latest version.
